.editorconfig

```
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.py]
+indent_size = 4 # pep8
```

```suggestion
[*.py]
indent_size = 4 # pep8
profile = black
```
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#formats

---

Found this this morning